README.md

```diff
@@ -6,7 +6,7 @@
 [![Twitter](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=Wow%20Check%20Loklak%20on%20@gitHub%20@loklak_app%20@loklak_:%20https://github.com/loklak/loklak_server%20&url=%5Bobject%20Object%5D)
 [![Twitter Follow](https://img.shields.io/twitter/follow/lklknt.svg?style=social&label=Follow&maxAge=2592000?style=flat-square)](https://twitter.com/loklak_)
 
-Development: [![Build Status](https://travis-ci.org/loklak/loklak_server.svg?branch=development)](https://travis-ci.org/loklak/loklak_server) [![Docker Build Status](https://img.shields.io/docker/build/jrottenberg/ffmpeg.svg)](https://hub.docker.com/r/loklak/loklak_server/builds/)
+Development: [![Build Status](https://travis-ci.org/loklak/loklak_server.svg?branch=development)](https://travis-ci.org/loklak/loklak_server) [![Docker Build Status](https://img.shields.io/docker/build/mariobehling/loklak.svg)](https://hub.docker.com/r/mariobehling/loklak/builds/)
```

Comment:
The repo is `loklak/loklak_server` and the docker images are no longer built on Dockerhub (they are built and pushed using Travis). I think we can remove the Docker tags.

Reply:
Removed the docker status tag